security-oauth2-authorization-server/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot'
+    id 'org.springframework.aot.smoke-test'
+    id 'org.graalvm.buildtools.native'
+}
+
+dependencies {
+    implementation(platform(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES))
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.security:spring-security-oauth2-authorization-server:1.0.0-M1")
+
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation("org.springframework.security:spring-security-test")
+
+    aotTestImplementation("org.springframework.boot:spring-boot-starter-json")
```

Comment:
I had to add this dependency because the `ObjectMapper` in the AOT tests were not being found.